cmd/kola/kola.go

```diff
@@ -250,12 +269,50 @@ func (i item) String() string {
 			}
 		}
 		i.Platforms = platforms
+	} else {
+		i.ExcludePlatforms = []string{""}
 	}
 	if len(i.Platforms) == 0 {
-		i.Platforms = []string{"all"}
+		if listJSON {
+			i.Platforms = kolaPlatforms
+		} else {
+			i.Platforms = []string{"all"}
+		}
 	}
 	if len(i.Architectures) == 0 {
-		i.Architectures = []string{"all"}
+		if listJSON {
+			i.Architectures = kolaArchitectures
+		} else {
+			i.Architectures = []string{"all"}
+		}
 	}
-	return fmt.Sprintf("%v\t%v\t%v", i.Name, i.Platforms, i.Architectures)
+	if len(i.ExcludeDistros) > 0 {
```

Comment:
Rather than copypasting this code, there could be a helper function (perhaps an anonymous one) that takes three string slices.